deployment/promotion/promote.py

```diff
@@ -9,396 +9,157 @@
 self-contained builds of the model version are found.
 - If you need to create a self-contained build of a model version, use the create_scb.py script.
 
-Configuration is done via environment variables. All are mandatory except VERTA_DEST_REGISTERED_MODEL:
+Configuration is done via environment variables. All are mandatory except VERTA_DEST_REGISTERED_MODEL_ID:
```

Comment:
Please add a comment somewhere up here specifying the minimum required client version to run this script.

Reply:
We don't have this information right now until we publish the client.

Reply:
We know what version should be next regardless of whether or not we've cut it yet(: